js_modules/dagit/packages/core/src/assets/AssetLink.tsx

```diff
@@ -33,6 +30,31 @@ export const AssetLink: React.FC<{
             )}
           {isGroup ? '/' : null}
         </span>
+      );
+    }
+
+    const assetPath = path
+      .reduce((accum, elem, ii) => {
+        return [...accum, ii > 0 ? ' / ' : '', elem];
+      }, [] as string[])
+      .join('');
```

Comment:
These changes all look good to me, but I'd be curious if this is visually the same as the big pile of react fragments rendered on 22-29. It kinda seems like all the spans + &nbsp/&nbsp; aren't really necessary and it could just re-use this string?